util/libcrypto.num

```diff
@@ -5197,3 +5197,17 @@ OSSL_DESERIALIZER_CTX_set_cleanup       ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_get_construct     ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_get_construct_data ?	3_0_0	EXIST::FUNCTION:
 OSSL_DESERIALIZER_CTX_get_cleanup       ?	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_CTX_set_tls1_prf_md            ?	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_CTX_set1_tls1_prf_secret       ?	3_0_0	EXIST::FUNCTION:
```

Comment:
I notice that EVP_PKEY_CTX_set_scrypt_N.pod has the following line
(line64) All the functions described here are implemented as macros.

Maybe this should be removed and described in a HISTORY section like we do elsewhere?

blah(), blah(),
were macros in OpenSSL 1.1.1 and below.
From OpenSSL 3.0 they are functions.

Reply:
Documentation updated.